### ISSUE TITLE
Training Freezed on custom dataset

### ISSUE THREAD

**Issue by sanjanasri** [awslabs/dgl-ke#216]
Hi all,

   I tried running "DGLBACKEND=pytorch dglke_train --model_name TransE_l1 --data_path ~/Memory --dataset Memory --data_files train.txt valid.txt test.txt --format raw_udd_hrt --save_path ~/Memory/model --batch_size 1000 --neg_sample_size 200 --hidden_dim 300 --gamma 19.9 --lr 0.25 --max_step 500 --log_interval 100 --batch_size_eval 16 -adv --regularization_coef 1.00E-09 --test --num_thread 1 --num_proc 8 " on the custom dataset

Training is freezer. Here is the screenshot for reference[
![Screenshot from 2021-06-02 22-02-54](https://user-images.githubusercontent.com/13214774/120517721-60e22c00-c3ee-11eb-80b1-7f5e93aba6ac.png)
](URL)

Thanks in advance

**Comment by classicsong** [comment]
Can you try run with single process first?

**Comment by sanjanasri** [comment]
@classicsong Thank your response. I tried. Still it is not training

**Comment by classicsong** [comment]
Does the batchsize larger than your edge numbers?

**Comment by sanjanasri** [comment]
@classicsong Thank you so much. I got the results after changing batch size. But the training accuracy is too low. my training triplets is 500, valid - 60 test - 30. Following is the command I followed



DGLBACKEND=pytorch dglke_train --model_name TransE_l1  --data_path ~/Memory --dataset Memory --data_files train.txt valid.txt test.txt --format raw_udd_hrt --save_path ~/Memory/model --batch_size 100 --neg_sample_size 10 --log_interval 100 --hidden_dim 300 --max_step 15000 --gamma 19.9 --lr 0.01 --regularization_coef 1.00E-09 --test  -adv

Result is 
[0]Test average MRR: 0.11240481223112556
[0]Test average MR: 414.96666666666664
[0]Test average HITS@1: 0.1
[0]Test average HITS@3: 0.11666666666666667
[0]Test average HITS@10: 0.11666666666666667
testing takes 0.035 seconds

**Comment by classicsong** [comment]
Your dataset is very slow. As dgl-ke training does not filter the false-negative samples during training, small dataset may have acc problems.

**Comment by sanjanasri** [comment]
@classicsong oh k ... Thank you so much..
